sim/mage/talents_fire.go

```diff
@@ -123,9 +123,10 @@ func (mage *Mage) applyHotStreak() {
 
 	ImprovedHotStreakProcChance := float64(mage.Talents.ImprovedHotStreak) * 0.5
 
-	// TODO: Is this supposed to be fully buffed Spell Crit % or only from gear Rating?
+	// Simcraft uses a reference from ElitistJerks that's no longer available, but their formula is
+	// max(0, -2.73 * player crit + 0.95)
 	baseCritPercent := mage.GetStat(stats.SpellCritPercent) + mage.GetStat(stats.CritRating)/core.CritRatingPerCritPercent
-	BaseHotStreakProcChance := float64(-2.7*baseCritPercent/100 + 0.9) // EJ settled on -2.7*critChance+0.9
+	mage.hotStreakProcChance = max(0, -2.73*baseCritPercent/100+0.95)
```

Comment:
Updated the Hot Streak calculation with values from simcraft

Reply:
You can probably link it to this post: https://web.archive.org/web/20120208064232/http://elitistjerks.com/f75/t110326-cataclysm_fire_mage_compendium/p6/#post1831143